src/components/capture/Interval/index.tsx

```diff
@@ -108,162 +78,69 @@ function CaptureInterval({ readOnly }: CaptureIntervalProps) {
-                                        color: (theme) =>
-                                            primaryButtonText[
-                                                theme.palette.mode
-                                            ],
-                                    },
-                                }}
-                                value={unit}
-                                variant="filled"
-                            >
-                                <MenuItem value="s">
-                                    <FormattedMessage id="captureInterval.input.seconds" />
-                                </MenuItem>
-
-                                <MenuItem value="m">
-                                    <FormattedMessage id="captureInterval.input.minutes" />
-                                </MenuItem>
-
-                                <MenuItem value="h">
-                                    <FormattedMessage id="captureInterval.input.hours" />
-                                </MenuItem>
-
-                                <MenuItem value="i">
-                                    <FormattedMessage id="captureInterval.input.interval" />
-                                </MenuItem>
-                            </Select>
-                        </InputAdornment>
-                    }
-                    error={errorsExist}
-                    id={INPUT_ID}
-                    label={label}
-                    onChange={(event) => {
-                        const value = event.target.value;
-
+                onChange={(_event, value) => {
+                    if (typeof value === 'string') {
                         setInput(value);
-                        updateStoredInterval(value, unit, setUnit);
-                    }}
-                    size={INPUT_SIZE}
-                    sx={{
-                        borderRadius: 3,
-                        pr: 0,
-                        width: 400,
-                    }}
-                    value={input}
-                />
+                        updateStoredInterval(value);
+                    }
+                }}
+                options={[
+                    '0s',
+                    '30s',
+                    '1m',
+                    '5m',
+                    '15m',
+                    '30m',
+                    '1h',
+                    '2h',
+                    '4h',
+                ]}
+                renderInput={({
+                    InputProps,
+                    ...params
+                }: AutocompleteRenderInputParams) => (
+                    <TextField
+                        {...params}
+                        InputProps={{
+                            ...InputProps,
+                            sx: { borderRadius: 3 },
+                        }}
+                        error={errorsExist}
+                        label={label}
+                        size="small"
+                        variant="outlined"
```

Comment:
This should probably match the other inputs and just use standard

Reply:
I am not a fan of the look of the standard input here. Dave will likely express the sentiment that the field does not appear to be interactive or is easily overlooked, but I will update it since that is your preference.